ios/PurchasesHybridCommon/PurchasesHybridCommon/CommonFunctionality.swift

```diff
@@ -268,27 +262,33 @@ import Purchases
 
     @objc(logOutWithCompletionBlock:)
     static func logOut(completion: @escaping ([String: Any]?, ErrorContainer?) -> Void) {
-        Purchases.shared.logOut(purchaserInfoCompletionBlock(from: completion))
+        Purchases.shared.logOut(completion: purchaserInfoCompletionBlock(from: completion))
     }
 
     @objc(createAlias:completionBlock:)
     static func createAlias(newAppUserID: String, completion: @escaping ([String: Any]?, ErrorContainer?) -> Void) {
-        Purchases.shared.createAlias(newAppUserID, purchaserInfoCompletionBlock(from: completion))
+        let hybridCompletion = purchaserInfoCompletionBlock(from: completion)
+        Purchases.shared.logIn(newAppUserID) { customerInfo, created, error in
+            hybridCompletion(customerInfo, error)
+        }
```

Comment:
I don't love this (remapping createAlias, identify and reset to identity v3) but it's only temporary until we update the public api

Reply:
i am assuming this is all done in the other PR, but if not, maybe TODOs would be good for places that need updating post-public-api-update?

Reply:
it's all removed in #131, but to-dos would have been better, yes